observatory-platform/observatory/platform/terraform/main.tf

```diff
@@ -347,6 +353,13 @@ resource "google_compute_firewall" "allow_ssh" {
   priority = 65534
 }
 
+resource "google_vpc_access_connector" "observatory_vpc_connector" {
+  name = local.vpc_connector_name
+  ip_cidr_range = "10.8.0.0/28"
+  network = local.network_name
+  region = var.google_cloud.region
+}
+
```

Comment:
Could you maybe elaborate a bit on what the vpc access connector does? It seems like this connects the cloud run backend to the vpc network, is this required for the added features of the API?

Reply:
The VPC Access Connector is required to enable the Cloud Run backend to connect to the CloudSQL database. I can add it as a comment.

Reply:
Ah thanks, I saw the added 'google_sql_database', but didn't check the 'google_sql_database_instance', where the network is specified, so I missed this.